lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/qm/XwikiCoreFacadeRestResource.java

```diff
@@ -92,8 +94,7 @@ public void genrationCompleted(String questionnairesId)
 		String uri = String.format("%s/learnpad/qm/corefacade/genrationcompleted/%s",
 				RestResource.REST_URI, questionnairesId);
 		PutMethod putMethod = new PutMethod(uri);
-		putMethod.addRequestHeader("Accept", "application/xml");
-
+		putMethod.addRequestHeader("Accept", MediaType.TEXT_PLAIN);
```

Comment:
Ha, I missed a part indeed. So even a response is not expected.  So it's not even necessary to have this line but hey, I'll merge it like that because it doesn't cause problem either.